app/src/components/checks.js

```diff
@@ -14,29 +13,28 @@ const checks = {
     };
 
     try {
-      const connection = new SMTPConnection({
+      const transporter = nodemailer.createTransport({
         host: host,
         port: 25,
         tls: {
-          rejectUnauthorized: false // Do not fail on invalid certs
+          rejectUnauthorized: false // do not fail on invalid certs
         }
       });
 
-      await connection.connect();
+      await transporter.verify();
       result.authenticated = true;
       result.authorized = true;
       result.healthCheck = true;
-      // connection.quit();
     } catch (error) {
       log.error('getSmtpStatus', error.message);
-      // connection.close();
     }
 
     return result;
   },
 
+  /** Returns a list of all endpoint connectivity states */
   getStatus: () => Promise.all([
-    checks.getSmtpStatus()
+    checks.getSmtpStatus('apps.smtp.gov.bc.cas') // TODO: move this to constants file
```

Comment:
apps.smtp.gov.bc.cas, typo, should be .ca?

Reply:
Whoops forgot to revert from testing.